IEEE-PSH/hackpsh-web
# tRPC & Proper Linting & Folder Restructure & Icon Integration / Release

# Description

* A new pattern has emerged for creating api-routes for specifically authenticated users only.
* The folder structure within src has been improved to show the three-tier architecture more clearly (db, server, app)
* Database queries are properly integrated with server-side tRPC routes and are type-safe
* Linting has not been working on the project since original setup, and has been properly updated. Please use the original prettier extension (updated as part of workspace recommendations).
* Initial DB Schema has been created to store a user's profile (holds all user information acquired from onboarding)
* Brand Logo has been created and integrated across theme and the whole site (Favicon)

# Test Guidelines

* Ensure existing functions (auth / theme) are stable on master
* Ensure deployment on all environments go smoothly (Local & Testing are stable, Testing on Master via PR)
